addons/storyshots/storyshots-puppeteer/src/axeTest.ts

```diff
@@ -17,7 +17,8 @@ export const axeTest = (customConfig: Partial<CommonConfig> = {}) =>
     ...customConfig,
     async testBody(page, options) {
       const parameters = options.context.parameters.a11y;
-      const include = parameters?.element ?? '#root';
+      const element = (parameters === null || parameters === undefined) ? undefined : parameters.element;
```

Comment:
This was the only place in the codebase that I could find that took advantage of the optional chaining and nullish coalescing operator introduced in 3.7.

I replaced it with the equivalent code here.